plugins/modules/zos_copy.py

```diff
@@ -2673,6 +2676,30 @@ def main():
                     sms_management_class=dict(type="str", required=False),
                 )
             ),
+            use_template=dict(type='bool', default=False),
```

Comment:
Shouldn't this have a 'required' entry in it?

Reply:
Are you asking because of the possibility that a user may set options in `template_parameters` while ignoring `use_template`? It may need a `required` entry, I need to verify that

Reply:
I think I misunderstood your comment yesterday. If I remember correctly, arguments need to have either a `required` entry or a `default` one. By using `default` the engine infers `required=False`.

Reply:
That's what I was asking about!  Okay, this looks okay to me.